pkg/tfbridge/tests/provider_test.go

```diff
@@ -285,444 +284,6 @@ func TestReproMinimalDiffCycle(t *testing.T) {
 	}`)
 }
 
-func TestValidateInputsPanic(t *testing.T) {
```

Comment:
Replaced with https://github.com/pulumi/pulumi-gcp/pull/2389 for now and will follow-up in https://github.com/pulumi/pulumi-terraform-bridge/issues/2418

The tests here only test the required behaviour indirectly through panics which no longer happen.

Reply:
It'd be nice if we had reliable coverage metrics to see that removing these tests doesn't decrease coverage %.

Reply:
It likely does. https://github.com/pulumi/pulumi-gcp/pull/2389 gives some coverage here. I plan to add back tests for the type checking warnings once I solve https://github.com/pulumi/pulumi-terraform-bridge/issues/2418